packages/sdk-client/package.json

```diff
@@ -0,0 +1,66 @@
+{
+  "name": "@commercetools/sdk-client-v2",
+  "version": "0.0.1",
+  "description": "commercetools TypeScript SDK client for Platform API",
+  "keywords": [
+    "commercetools",
+    "sdk",
+    "typescript",
+    "client",
+    "middleware",
+    "http",
+    "oauth",
+    "auth"
+  ],
+  "homepage": "https://github.com/commercetools/commercetools-sdk-typescript",
+  "license": "MIT",
+  "directories": {
+    "lib": "lib",
+    "test": "test"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/commercetools/commercetools-sdk-typescript.git"
+  },
+  "bugs": {
+    "url": "https://github.com/commercetools/commercetools-sdk-typescript/issues"
+  },
+  "dependencies": {
+    "@commercetools/platform-sdk": "^1.17.1",
```

Comment:
This should not be a dependency as platform SDK is dependent on the client package. Else you run in a circular dependency

Reply:
We need this for a CI build before we can run tests, if I should remove it, the build will fail exit, hence test won't even run.